### PR TITLE
app-arch/createrepo_c: new package

### PR DESCRIPTION
**Pull request by jkroonza**
Signed-off-by: Jaco Kroon <jaco@uls.co.za>

### CODE REVIEW

**Comment by jkroonza** [comment]
Should fix python first ...

File: app-arch/createrepo_c/createrepo_c-0.16.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+# Notes:  Help with enabling the python support would be great.
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="C implementation of createrepo"
+HOMEPAGE="https://github.com/rpm-software-management/createrepo_c"
+SRC_URI="https://github.com/rpm-software-management/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
```
Comment (DarthGandalf) [inline app-arch/createrepo_c/createrepo_c-0.16.1.ebuild:11]: No PN please

https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Plaintexting_.24.7BPN.7D

File: app-arch/createrepo_c/createrepo_c-0.16.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="app-arch/bzip2
+app-arch/rpm
```
Comment (DarthGandalf) [inline app-arch/createrepo_c/createrepo_c-0.16.1.ebuild:18]: please indent

File: app-arch/createrepo_c/createrepo_c-0.16.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+DESCRIPTION="C implementation of createrepo"
+HOMEPAGE="https://github.com/rpm-software-management/createrepo_c"
+SRC_URI="https://github.com/rpm-software-management/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="app-arch/bzip2
```
Comment (DarthGandalf) [inline app-arch/createrepo_c/createrepo_c-0.16.1.ebuild:17]: This one needs :=

File: app-arch/createrepo_c/createrepo_c-0.16.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+app-arch/rpm
+dev-db/sqlite:3
+dev-libs/glib:2
+dev-libs/libxml2
+dev-libs/openssl
```
Comment (DarthGandalf) [inline app-arch/createrepo_c/createrepo_c-0.16.1.ebuild:22]: :=

**Pull request closed without merging**